pages/documentation/showcase.md

```diff
@@ -159,7 +159,7 @@ If your project is currently using USWDS and you would like to see it included i
 - [USA.gov](https://usa.gov)
 - [USAID](https://www.usaid.gov/)
 - [USAJOBS](https://www.usajobs.gov/)
-- [USAJOBS Data Science Careers](https://www.usajobs.gov/careerfields/data-science)
+- [USAJOBS Data Science Careers](https://data-science.usajobs.gov/)
```

Comment:
@amyleadem @mahoneycm - Can you check this link as well just to confirm this is the correct new URL for the intended page? Received a new test failure for this link.

Reply:
Lgtm! Compared on Wayback Machine

Reply:
> [!note]
> Confirming that this page still appears to be using USWDS (as much as it was before). I believe this meets the criteria for the showcase.
> 
> [Before](https://web.archive.org/web/20230102223425/https://www.usajobs.gov/careerfields/data-science/) | [After](https://data-science.usajobs.gov/)